test/e2eTests/pages/clinicPage.js

```diff
@@ -0,0 +1,9 @@
+const Page = require('./page')
+
+module.exports = new Page({
+  isLoaded: {
+    element: '[ng-app="clinic"]',
+    urlPart: '/clinic',
+  },
+  components: ['patientSearch'],
```

Comment:
This is how a page defines its components. In effect, this copies functionality from the ```PatientSearchComponent``` onto the clinic page object, which allows tests to call ```clinicPage.search()```